SR2023/2023-01-19-competition-venue-announcement.md

```diff
@@ -25,9 +25,14 @@ All events can be found on our website: https://studentrobotics.org/events/
 
 ## Simulator
 
-We've also just released a new version of the simulator: `2023.2`. This includes the ability to grab gold tokens and add the `Robot.is_simulated()` method.
+We've also just released a new version of the simulator: `2023.2`. This includes the following changes:
 
-Full release notes and a download link can be found [on the docs](https://studentrobotics.org/docs/simulator/).
+- widens and raises the robot grabber so that it can pick up gold tokens; this also changes some of the sensors to account for the possible gold token in the front of the robot:
+   - the camera is now higher up, tilted slightly downwards and has a wider field of view
+   - the front ultrasounds sensors are moved forward slightly
+- adds `Robot.is_simulated()` to detect that code is running in the simulator
+
+This release can be found [on the docs](https://studentrobotics.org/docs/simulator/).
```

Comment:
Maybe link direct to the section? (https://studentrobotics.org/docs/simulator/#installing-the-simulation)